src/Shared/RoslynUtils/SymbolExtensions.cs

```diff
@@ -145,4 +186,17 @@ public static bool TryGetNamedArgumentValue<T>(this AttributeData attribute, str
         }
         return false;
     }
+
+    public static string GetParameterInfoFromConstructor(this IParameterSymbol parameterSymbol)
```

Comment:
Maybe this follows an established pattern, but I couldn't tell from the name or signature that this returned a blob of code.

Reply:
I tweaked the name here to indicate it returns code.